docs/src/content/docs/guides/folder-structure.mdx

```diff
@@ -37,13 +37,13 @@ The ProofKit CLI expects a certain structure for it to be able to inject code in
 
 ## The `src` directory
 
-This is where your application code lives. Files outside of the `src` folder are for configuation only.
+This is where your application code lives. Files outside of the `src` folder are for configuration only.
 
 ## The `src/app` directory
 
-This is the Next.js app router. Learn more at the [Next.js docs](https://nextjs.org/docs/app/building-your-application/routing), and make sure you're looking at the guides for "using the app router".
+This is the Next.js app router. Learn more at the [Next.js docs](https://nextjs.org/docs/app/building-your-application/routing), specifically the section that discusses  "using the app router".
```

Comment:
If you look at the nextjs docs, there is really 2 "modes" to view them in:  "using the app router" or "using the pages router" so I dont' think if "section" is the right term here

If you change the "mode" of the docs, the entire sidebar changes...

![CleanShot 2024-11-18 at 16.48.01@2x.png](https://graphite-user-uploaded-assets-prod.s3.amazonaws.com/hjFfXocKfkN5SpCM7evU/d9a5f175-8852-46e3-9946-4f609334974f.png)

Reply:
how about, "...specifically "using the app router"."

Reply:
if in quotes it should say "Using App Router", but sure